### DISCUSSION TITLE
[灌水]我是不是脸太黑了。。。

### DISCUSSION THREAD

**Discussion by nninng** [share121/inter-knot#2965] [其他]
![Uploading IMG_20240818_232832.jpg…]()

在报刊亭刮刮刮卡，连续两天刮到三根骨头会有一个成就，叫“**该洗脸了**”，别问我怎么知道的，我的脸难道有这么黑吗。。。

**Comment by LuAn-afl** [comment]
抽卡不黑就<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
